android/src/main/java/com/svtek/reactnative/RNAndroidSharerModule.java

```diff
@@ -62,15 +62,45 @@ private String getMimeType(String filePath) {
     }
   }
 
+  private boolean checkAppInstall(String uri) {
+    PackageManager pm = this.reactContext.getPackageManager();
+    try {
+        pm.getApplicationInfo(uri, 0);
+        return true;
+    } catch (PackageManager.NameNotFoundException e) {
+        // Error
+    }
```

Comment:
Please use two spaces as tabs. Also correct all tab inconsistencies in this commit.

Reply:
All tab inconsistencies fixed by Android Studio Reformat Code.